loky/backend/semaphore_tracker.py

```diff
@@ -71,6 +74,7 @@ def ensure_running(self):
                 self._fd = None
                 self._pid = None
 
+                warnings.filterwarnings('always', module=__name__)
```

Comment:
We should not change the warning filters in the implementation. Otherwise the user cannot control which warnings they want silenced or not. We should only do that in the tests.